react/features/filmstrip/components/native/Filmstrip.js

```diff
@@ -180,42 +170,18 @@ class Filmstrip extends Component<Props> {
  *
  * @param {Object} state - The redux state.
  * @private
- * @returns {{
- *     _participants: Participant[],
- *     _visible: boolean
- * }}
+ * @returns {Props}
  */
 function _mapStateToProps(state) {
     const participants = state['features/base/participants'];
     const { enabled } = state['features/filmstrip'];
 
     return {
-        /**
-         * The indicator which determines whether the filmstrip is enabled.
-         *
-         * @private
-         * @type {boolean}
-         */
+        _aspectRatio: state['features/base/responsive-ui'].aspectRatio,
```

Comment:
Same as above. The less updates the better, I think. I know it's not heavy, but if we can do better, then why not?

This is also a reason why I always try to stay away from having ```conference``` as a redux prop here, because that triggers a sh*t load of updates.

Reply:
Same, this is a literal.